src/core/operations/ToByteStringLiteral.mjs

```diff
@@ -0,0 +1,140 @@
+        this.inputType = "ArrayBuffer" ;
+        this.outputType = "string";this.infoURL = "https://en.wikipedia.org/wiki/String_(computer_science)#Non-text_strings";
+
+        this.args = [
+            {
+                "name": "Language",
+                "type": "option",
+                "value": Object.keys(LANGUAGES)
+            },
+        ];
+    }
+
+    /**
+     * @param {ArrayBuffer} input
+     * @param {Object[]} args
+     * @returns {string}
+     */
+    run(input, args) {
+        const data = new Uint8Array(input);
+        const language = LANGUAGES[args[0]];
+        if (language === "c") {
+            const sequences = Object.assign(DOUBLEQUOTE_SEQUENCE, COMMON_SEQUENCES, C_EXTRA_SEQUENCES);
+            // regex is here to replace \xa7a by \xa7""a since escape sequence can have more than 2 digit
+            return '"' + this.escape(data, sequences).replace(/(\\x[0-9a-f]{2})([0-9a-f])/gi, '$1""$2') + '"';
+        } else if (language === "go") {
+            const sequences = Object.assign(DOUBLEQUOTE_SEQUENCE, COMMON_SEQUENCES, GO_EXTRA_SEQUENCES);
+            return '([]byte)("' + this.escape(data, sequences) + '")';
+        } else if (language === "python") {
+            const [quote, quoteSequence] = this.preferedQuote(data);
+            const sequences = Object.assign(quoteSequence, COMMON_SEQUENCES, PYTHON_EXTRA_SEQUENCES);
+            return "b" + quote + this.escape(data, sequences) + quote;
+        }
+        return "";
+    }
+
+    /**
+     * @param {Uint8Array} data
+     * @returns {bool}
+     * python and javascript can use single or double quote equally
+     * better use the variant that reduce quote escape
+     */
+    preferedQuote(data) {
+        let onlySingleQuoteInData = false;
+        for (let i = 0; i < data.length; i++) {
+            if (data[i] === '"'.charCodeAt(0)) {
+                onlySingleQuoteInData = false;
+                break;
+            }
+            if (data[i] === "'".charCodeAt(0)) {
+                onlySingleQuoteInData = true;
+            }
+        }
+        if (onlySingleQuoteInData) {
+            return ['"', DOUBLEQUOTE_SEQUENCE];
+        }
+        return ["'", SINGLEQUOTE_SEQUENCE];
+    }
+
+    /**
+     * @param {Uint8Array} data
+     * @param {object} sequences
+     * @returns {string}
+     */
+    escape(data, sequences) {
+        let output = "";
+        for (let i = 0; i < data.length; i++) {
+            output +=
+                sequences[data[i]] ||
+                (
+                    (data[i] < 0x20 || data[i] > 0x7e) ?
+                        "\\x" + data[i].toString(16).padStart(2, 0) :
+                        String.fromCharCode(data[i])
+                );
+        }
+        return output;
+    }
+}
+
+const SINGLEQUOTE_SEQUENCE = {
```

Comment:
I would prefer all the `const` declarations together at the top of the file, please

Reply:
done